lute/require/include/lute/bundlevfs.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "lute/modulepath.h"
+
+#include <optional>
+#include <string>
+#include <map>
```

Comment:
```suggestion
#include "Luau/DenseHash.h"
```
Is it possible to replace `std::map` with Luau's dense hash map?